test/course_planner/events/events_test.exs

```diff
@@ -21,6 +21,20 @@ defmodule CoursePlanner.EventsTest do
       assert Events.all_with_users() == [event |> Repo.preload(:users)]
     end
 
+    test "all_splitted/1 returns events splited by time" do
+      %{id: id1} = insert(:event, %{date: ~D[2017-02-02], starting_time: ~T[10:00:00.000000]})
```

Comment:
please use Timex for setting time in future or past or your test becomes obsolete after a while
E.g. `Timex.shift(Timex.now(), days: -10))`